ee/app/livechat-enterprise/server/settings.js

```diff
@@ -115,6 +63,93 @@ export const createSettings = () => {
 				],
 			});
 		});
+
+		this.section('Queue_management', function() {
+			this.add('Livechat_waiting_queue', false, {
+				type: 'boolean',
+				group: 'Omnichannel',
+				section: 'Queue_management',
+				i18nLabel: 'Waiting_queue',
+				enterprise: true,
+				invalidValue: false,
+			});
+
+			this.add('Livechat_waiting_queue_message', '', {
+				type: 'string',
+				group: 'Omnichannel',
+				section: 'Queue_management',
+				i18nLabel: 'Waiting_queue_message',
+				i18nDescription: 'Waiting_queue_message_description',
+				enableQuery: { _id: 'Livechat_waiting_queue', value: true },
+				enterprise: true,
+				invalidValue: '',
+				modules: [
+					'livechat-enterprise',
+				],
+			});
+
+			this.add('Livechat_maximum_chats_per_agent', 0, {
+				type: 'int',
+				group: 'Omnichannel',
+				section: 'Queue_management',
+				i18nLabel: 'Max_number_of_chats_per_agent',
+				i18nDescription: 'Max_number_of_chats_per_agent_description',
+				enableQuery: { _id: 'Livechat_waiting_queue', value: true },
+				enterprise: true,
+				invalidValue: 0,
+				modules: [
+					'livechat-enterprise',
+				],
+			});
+
+			this.add('Livechat_number_most_recent_chats_estimate_wait_time', 100, {
+				type: 'int',
+				group: 'Omnichannel',
+				section: 'Queue_management',
+				i18nLabel: 'Number_of_most_recent_chats_estimate_wait_time',
+				i18nDescription: 'Number_of_most_recent_chats_estimate_wait_time_description',
+				enableQuery: { _id: 'Livechat_waiting_queue', value: true },
+				enterprise: true,
+				invalidValue: 100,
+				modules: [
+					'livechat-enterprise',
+				],
+			});
+
+			this.add('Livechat_max_queue_wait_time_action', 'Nothing', {
+				type: 'select',
+				group: 'Omnichannel',
+				section: 'Queue_management',
+				i18nLabel: 'Livechat_max_queue_wait_time_action',
+				i18nDescription: 'Livechat_max_queue_wait_time_action',
```

Comment:
You are displaying the setting label as setting description, is that on purpose?

Reply:
Nope, I removed it 😬